Index: Front End/EbacWeb/src/main/webapp/jsp/module/advicemanager/confirmAdviceManagerConfig.jsp
===================================================================
--- Front End/EbacWeb/src/main/webapp/jsp/module/advicemanager/confirmAdviceManagerConfig.jsp	(revision 108560)
+++ Front End/EbacWeb/src/main/webapp/jsp/module/advicemanager/confirmAdviceManagerConfig.jsp	(working copy)
@@ -350,11 +350,7 @@
 		</bac:validateAccess>
 		<!--Fin tabla Configuracion de alertas de ingreso por horario-->
 
-	<tiles:insert definition="show.dfa.ajax">
-		<tiles:put name="functionality" value="MANDEV" />
-	</tiles:insert>
-
-	<form name="frmAdviceManager" action="" method="POST">
+	<form name="frmAdviceManager" action="/ebac/module/advicemanager/activateAdviceManager.go" method="POST">
 			<div class="bel-grid-row">
 				<div class="bel-col-12 bel-btn-group bel-space-top-l">
 					<button class="bel-btn bel-btn-primary bel-btn-primary-active"
@@ -397,8 +393,5 @@
 			thAlign : [ "left", "left", "left" ],
 			tdAlign : [ "left", "left", "left" ]
 		});
-
-		/** DFA */
-		$BLUEJQuery.dfa.init();
 });
 </script>
